src/adapters/streaming-ttl-checker.ts

```diff
@@ -0,0 +1,70 @@
+import { AppComponents } from '../types'
+import { IStreamingChecker } from '../types/checker.type'
+import { CronJob } from 'cron'
+
+export async function createStreamingTTLChecker(
+  components: Pick<AppComponents, 'logs' | 'sceneStreamAccessManager' | 'livekit'>
+): Promise<IStreamingChecker> {
+  const { logs, sceneStreamAccessManager, livekit } = components
+  const logger = logs.getLogger(`streaming-ttl-checker`)
+  let job: CronJob
+  async function start(): Promise<void> {
+    job = new CronJob(
+      '* * * * *',
```

Comment:
Will CronJob launch another job if the one before it hasn't finished? The time the cron-job runs must be taken into consideration in order not to end up running concurrently with another job.

Reply:
what about a flag? check it again please

Reply:
The flag works, yes!